src/plugins/wizard/tsconfig.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Can we not commit this file?

Reply:
This file is used by the scripts/type_check script to run tsc localized to a single file.  Other plugins have this file: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/opensearch_dashboards_react/tsconfig.json